#42: Add documentation on excludes (#44)
exasol/error-code-crawler-maven-plugin: README.md

For the moment the value of this property needs to be updated manually

### Excludes

> In some very rare cases you may want to exclude some files from crawling. But usually: Don't do it!

Reviewer: ```suggestion
In some very rare cases you may want to exclude some files from crawling. But we don't recommend excluding files.
```

You can define the excludes by adding the following configuration to the maven-plugin:

```xml

<configuration>
    <excludes>
        <exclude>**/MyTest.java</exclude>
    </excludes>
</configuration>
```

> The excludes only affect the validation – not compiling. So if the specific file has syntax errors, excluding won't help.

Reviewer: ```suggestion
The excludes only affect the validation &mdash; not compiling. So if the specific file has syntax errors, excluding won't help.
```
